lib/manager/util/transform.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import toSentenceCase from '../../common/util/to-sentence-case'
+
+import type {FeedTransformation, FeedVersion} from '../../types'
+
+/**
+ * Returns human readable name for transformation type.
+ */
+export function getTransformationName (
```

Comment:
This function seems very susceptible to funky additions and odd naming patterns for each transformation type. Already, I don't think this would work with a `DeleteRecordsTransformation` type. I think it'd be a little more maintainable if we used the strategy pattern and had each transformation type have it's own function to create the transformation name. Each of these functions could be composed of smaller sub-functions if needed.

Reply:
Also, it'd be great for this function to have a unit test.